packages/server/src/course/course.entity.ts

```diff
@@ -28,7 +28,10 @@ export class CourseModel extends BaseEntity {
   queues: QueueModel[];
 
   @Column('text')
-  name: string;
+  name: string; // display name entered by Prof
+
+  @Column('text', { nullable: true }) // nullable for backwards compatibility
+  sectionGroupName: string; // from admin
```

Comment:
Should we write a backfill to make old courses have the same name and question group name?

Reply:
Hmm I can log a ticket to come back to this once the other login stuff is done? That's probably nice to have but idt we actually will use the sectionGroupName for anything immediately so it might be fine if it's just null :P

#793